pkg/security/ebpf/c/include/hooks/module.h

```diff
@@ -50,6 +50,22 @@ int __attribute__((always_inline)) trace_kernel_file(struct pt_regs *ctx, struct
     return 0;
 }
 
+SEC("kprobe/parse_args")
+int kprobe_parse_args(struct pt_regs *ctx){
+    char *args = (char *) PT_REGS_PARM2(ctx);
+    u32 isTruncated = 0;
+    struct syscall_cache_t *syscall = peek_syscall(EVENT_INIT_MODULE);
+    if (!syscall) {
+        return 0;
+    }
+    int len = bpf_probe_read_str(&syscall->init_module.args, sizeof(syscall->init_module.args)+1, args);
+    if (len==129){
+        isTruncated = 1;
+    }
+    bpf_probe_read(&syscall->init_module.argsTruncated,sizeof(syscall->init_module.argsTruncated),&isTruncated);
```

Comment:
`bpf_probe_read` isn't needed here, you can directly copy the variable like this: `syscall->init_module.args_truncated = is_truncated`